core-toolbar.css

```diff
@@ -87,20 +111,24 @@ polyfill-next-selector { content: ':host.core-narrow .toolbar-tools, .core-narro
   left: 0;
 }
 
+/* ******************************* */
+/* TODO: Should content be styled? */
```

Comment:
I think is important when you put some buttons inside the toolbar those buttons will automatically have the right paddings/margins so they look good.  Also we can consider dropping core-toolbar and just make a material design version and the stylings should follow the [material design spec](http://www.google.com/design/spec/layout/structure.html#structure-app-bar).